lib/src/Pos/Logic/Types.hs

```diff
@@ -27,47 +26,33 @@ import           Pos.Util.Chrono (NE, NewestFirst, OldestFirst)
 -- | The interface to a logic layer, i.e. some component which encapsulates
 -- blockchain / crypto logic.
 data Logic m = Logic
-    { -- The stakeholder id of our node.
+    { -- | The stakeholder id of our node.
       ourStakeholderId   :: StakeholderId
-      -- Get a block, perhaps from a database.
+      -- | Get a block, perhaps from a database.
     , getBlock           :: HeaderHash -> m (Maybe Block)
-      -- Stream blocks from first hash to second hash.
-      -- Conduit is chosen mainly due to precedent: it's already used in
-      -- cardano-sl.
-    , getChainFrom       :: HeaderHash
-                         -> Source m Block
```

Comment:
Why did you delete this? Please restore it.

Reply:
Oh

> removes getChainFrom (which must be implemented in a completely different way and wasn't used anywhere).

It's going to be used. CSL-2148

Reply:
Yes, my idea is that:
1. All DB traversals should be conduits (it's easy to do it).
2. `getHeadersFromManyTo` and `getHashesRange` should be too, also I think first one should be implemented in terms of the second one. 

So `getChainFrom` is an unused code with implementation that will be totally different, that's why i deleted it. I suggest we keep things simple and move on incrementally. It's 5m to somebody to return it back, but it takes much more time for me to figure out which functionality is actually needed and which is unused.